app/assets/scripts/components/spotlight/single/index.js

```diff
@@ -312,10 +312,15 @@ SpotlightAreasSingle.propTypes = {
 function mapStateToProps (state, props) {
   const { spotlightId } = props.match.params;
 
+  const spotlightList = wrapApiResult(state.spotlight.list)
+
+  const site = spotlightList.getData().find(x => x.id === spotlightId)
```

Comment:
instead of using a value from the hardcoded summaries.js file, use the `summaries` field from the API response.

Reply:
Not knowing much about this code base, do we have some tests in place to verify this change works as expected?

Reply:
Unfortunately, there aren't any existing tests. I started to add some, and found that package.json has the sad action: `"test": "echo \"No tests\" && exit 0"` ☹️ I could add some, but JS/React is not really in my wheelhouse, so it would probably take a while to figure it out.